Backpack/src/main/java/net/skyscanner/backpack/util/Canvas.kt

```diff
@@ -0,0 +1,9 @@
+package net.skyscanner.backpack.util
```

Comment:
Extensions is sort of utils. I'd like to avoid confusion here as we might want to include some extensions to out public API in the future.

Reply:
Not gonna block the PR because of this but `utils` is too generic. Anything can be considered util if we follow this approach, which is not good, I think it's better to restrict  "global" extensions like this to one package, e.g. `extensions`